ddtrace/internal/constants.py

```diff
@@ -9,13 +9,15 @@
 PROPAGATION_STYLE_B3_SINGLE = "b3"
 _PROPAGATION_STYLE_W3C_TRACECONTEXT = "tracecontext"
 _PROPAGATION_STYLE_NONE = "none"
-_PROPAGATION_STYLE_DEFAULT = "datadog,tracecontext"
+_PROPAGATION_STYLE_DEFAULT = "datadog,tracecontext,baggage"
```

Comment:
Do we need to make this change? Baggage shouldn't be used to propagate distributed tracing headers. Adding it to propagation styles looks a bit odd

Reply:
The RFC says "The default values for these settings will change to include the baggage propagator: DD_TRACE_PROPAGATION_STYLE=datadog,tracecontext,baggage" So I was going off based on that

Reply:
It was suggested by @zacharycmontoya (and I agree) that we follow OpenTelemetry here, where the baggage propagator is another propagator that is enabled/disabled using `OTEL_PROPAGATORS`, along with `tracecontext`, `b3`, etc.

See https://opentelemetry.io/docs/languages/sdk-configuration/general/#otel_propagators